src/Stripe.net/Services/Customers/StripeCustomerCreateOptions.cs

```diff
@@ -10,12 +10,18 @@ public class StripeCustomerCreateOptions
         [JsonProperty("account_balance")]
         public int? AccountBalance { get; set; }
 
+        [JsonProperty("billing")]
```

Comment:
This is a new feature. IMO we should not support this on Custom creation/update. We should only support this on the Subscription resource. That's what we do in stripe-go